client/src/components/organisms/style/AdminRequestGroupBrowser.scss

```diff
@@ -0,0 +1,32 @@
+.request-group-description {
+    margin-left: 145px;
+}
+
+.btn, .btn:hover, .btn:focus, .btn:active, .show > .btn-primary.dropdown-toggle {
+    background-color: #fbfbfb;
+    color: #004D57;
+    border-color: #004D57;
+    height: 36px;
```

Comment:
please replace all the colors in this file with variables from `_config.scss`

Reply:
Done